src/main/java/br/com/openenade/api/estado/EstadoUtils.java

```diff
@@ -0,0 +1,14 @@
+package br.com.openenade.api.estado;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+public class EstadoUtils {
```

Comment:
@javanlacerda this is the same code of AnoUtils, please do not duplicate the code!